manifests/plugin/remote_execution/ssh.pp

```diff
@@ -63,4 +63,17 @@
       creates => $ssh_identity_path,
     }
   }
+
+  case $::osfamily {
+    'Debian': {
+      $core_package = 'ruby-smart-proxy-remote-execution-ssh-core'
+    }
+    default: {
+      $core_package = 'tfm-rubygem-smart_proxy_remote_execution_ssh_core'
+    }
+  }
+
+  package { $core_package:
+    ensure => installed',
+  }
```

Comment:
@ekohl I would take it more as an external dependency: it's really running a separate process outside of the proxy service itself. I don't think the SCL is important here. I think it would be more confusing pretending it the plugin is `$scl`-ized, as the plugin itself still runs in non-scl, it just proxies the requests to another process to do the actual work, but the proxy itself doesn't care, if it's running in SCL or not.